cmake/pcl_find_libusb.cmake

```diff
@@ -0,0 +1,25 @@
+#
+#pcl_find_libusb is used due to VCPKG making impossible to use local findXX modules.
+# and VCPKG findlibusb doesn't create the libusb targets.
+
+# Find and set libusb
+find_package(libusb)
+
+if(TARGET libusb::libusb)
+  #libusb target is found by the find_package script. 
+  #VCPKG skip PCLs findlibusb and sets its own variables which is handled below.
+  return()
+endif()
+
+#Handle VCPKG definitions
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(LIBUSB DEFAULT_MSG LIBUSB_LIBRARIES LIBUSB_INCLUDE_DIRS)
+
+mark_as_advanced(LIBUSB_INCLUDE_DIRS LIBUSB_LIBRARIES)
+
+if(LIBUSB_FOUND)
+  add_library(libusb::libusb UNKNOWN IMPORTED)
+  set_target_properties(libusb::libusb PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${LIBUSB_INCLUDE_DIRS}")  
+  set_target_properties(libusb::libusb PROPERTIES IMPORTED_LOCATION_DEBUG "${LIBUSB_LIBRARY_DEBUG}")
```

Comment:
@larshg: Do you know how CMaKe handles it, when you set an empty value here? Or do you have to wrap this line into a condition which checks if debug is present?

Reply:
I don't know it for sure, but I assume that if its a empty string you will just not have linked or added the correct include folder, hence your build will fail in one or the other way.
However, LIBUSB_FOUND will not be true if at least LIBUSB_INCLUDE_DIRS and LIBUSB_LIBRARIES are empty.

Reply:
The question was: What happens, when `LIBUSB_LIBRARY_DEBUG` is empty, but `LIBUSB_LIBRARY_RELEASE` not - does the debug build using the release value than or an empty value? ;-)

Reply:
So would you suggest to wrap this `set_target_properties` statement in an `if()`, checking if `LIBUSB_LIBRARY_DEBUG` is not empty, to be safe?

Reply:
I searched now in the module directory of CMake how they handle `IMPORTED_LOCATION_DEBUG`. It seems they are using always this 3 cases:

https://gitlab.kitware.com/cmake/cmake/-/blob/v3.22.0/Modules/FindJPEG.cmake#L120-L138

Reply:
So I'll add this as well as something similar to the Qhull branch.

Reply:
Implemented.